src/Compilers/Core/CodeAnalysisTest/EditorConfigTests.cs

```diff
@@ -262,50 +262,50 @@ public void ReservedKeys()
         public void SimpleNameMatch()
         {
             string regex = EditorConfig.TryCompileSectionNameToRegEx("abc");
-            Assert.Equal("^abc$", regex);
+            Assert.Equal("^.*/abc$", regex);
 
-            Assert.Matches(regex, "abc");
-            Assert.DoesNotMatch(regex, "aabc");
-            Assert.DoesNotMatch(regex, " abc");
-            Assert.DoesNotMatch(regex, "cabc");
+            Assert.Matches(regex, "/abc");
+            Assert.DoesNotMatch(regex, "/aabc");
+            Assert.DoesNotMatch(regex, "/ abc");
+            Assert.DoesNotMatch(regex, "/cabc");
         }
 
         [Fact]
         public void StarOnlyMatch()
         {
             string regex = EditorConfig.TryCompileSectionNameToRegEx("*");
-            Assert.Equal("^[^/]*$", regex);
+            Assert.Equal("^.*/[^/]*$", regex);
 
-            Assert.Matches(regex, "abc");
-            Assert.Matches(regex, "123");
-            Assert.DoesNotMatch(regex, "abc/123");
+            Assert.Matches(regex, "/abc");
+            Assert.Matches(regex, "/123");
+            Assert.Matches(regex, "/abc/123");
         }
 
         [Fact]
         public void StarNameMatch()
         {
             string regex = EditorConfig.TryCompileSectionNameToRegEx("*.cs");
-            Assert.Equal("^[^/]*.cs$", regex);
+            Assert.Equal("^.*/[^/]*.cs$", regex);
```

Comment:
❓ Why not `^.*/[^/]*\.cs$`?

Reply:
I fixed this in a later PR, but I might as well bring that change in with this one. I'll update the PR.